packages/playground/src/index.css

```
@@ -154,12 +154,87 @@ th:has(header) {
+    position: relative;
+    overflow: hidden;
+    isolation: isolate;
+  }
+
+  /* Ambient aurora — agent-tinted, slow, always present, low contrast. */
+  .ready-stage::before {
+    content: '';
+    position: absolute;
+    inset: -25%;
+    background:
+      radial-gradient(ellipse 50% 45% at 30% 28%, var(--ready-aurora-1, rgba(212, 212, 216, 0.14)), transparent 62%),
```

Stylelint (17.12.0)

[error] Expected "0.14" to be "14%" (alpha-value-notation)

(alpha-value-notation)